src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        Lottery lot = new Lottery();

```

Comment:
Please be more accurate with naming variables. We shouldn't  cut and abbreviate the words instead of using whole words. Take a look at examples in the course materials.